ext/standard/basic_functions.stub.php

```diff
@@ -1034,3 +1234,23 @@ function mt_getrandmax(): int {}
 function random_bytes(int $length): string {}
 
 function random_int(int $min, int $max): int {}
+
+/* soundex.c */
+
+function soundex(string $string): string|false {}
+
+/* uniqid.c */
+
+#ifdef HAVE_GETTIMEOFDAY
+function uniqid(string $prefix = "", bool $more_entropy = false): string {}
+#endif
+
+/* uuencode.c */
+
+function convert_uuencode(string $data): string|false {}
+
+function convert_uudecode(string $data): string|false {}
+
+/* versioning.c */
+
+function version_compare(string $version1, string $version2, string $operator = UNKNOWN): int|bool|null {}
```

Comment:
In the future, we might want to make the null case throw: It looks like right now we're not even generating a warning if you pass an incorrect comparison operator.

Reply:
Yep, agreed